### DISCUSSION TITLE
Tweak PR title/Commit message for multiple dependency updates

### DISCUSSION THREAD

**Discussion by EppO** [renovatebot/renovate#27926] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Currently, with the config I posted below
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "packageRules": [
    {
      "matchManagers": ["terraform"],
      "groupName": "{{parentDir}}",
      "commitMessageTopic": "Terraform {{{replace 'myhostedgitlab\/terraform\/([a-z\-]*)\/azurerm' '$1 module' depName}}}",
      "commitMessagePrefix": "{{parentDir}} - "
    }
  ],
}
```

When a single dependency is getting updated, the PR title/commit message is as follow:
```
dirA - Update Terraform storage module to v2.0.0
```

When multiple dependencies are getting updated, the PR title/commit message is as follow:
```
dirB - Update dirB
```

Is there a way to edit the commit message when multiple dependencies are being updated in the same PR? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
The title is like it is because of your configuration. Ie you've set those values you see. 

If you're sure that grouping is the approach you want then consider leveraging groupSlug so that groupName can be longer / more explanatory. 

Avoid editing commitMessageX if you can

**Reply by EppO** [reply]
You mean groupName is used when multiple dependencies are set?
I use grouping because I would like to open one single PR per folder containing Terraform environments, not one PR per module.

**Reply by EppO** [reply]
I just tried using groupSlug like this:
```
 "groupName": "Terraform module updates",
 "groupSlug": "{{parentDir}}"
```
but it generates a branch name called `renovate/parentdir`, seems like I can't use `parentDir` with groupSlug :(

**Reply by rarkins** [reply]
I think that could be fixed

**Reply by EppO** [reply, marked as answer]
I can open a feature request for that
EDIT: actually I see you already implemented it: #27927, thanks!!!
